cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -343,7 +344,8 @@ data:
       matches [{ "_SYSTEMD_UNIT": "docker.service" }]
       <storage>
         @type local
-        persistent true
```

Comment:
why do you remove persistence? 
its there to be able to pickup the buffer after a container crash.
otherwise you might loose some logs.

Reply:
I don't found lose some logs, but when config persitent true, disk io on node will be very high